superset/dashboards/api.py

```diff
@@ -993,8 +999,17 @@ def import_(self) -> Response:
         )
         overwrite = request.form.get("overwrite") == "true"
 
+        config_overwrite: Dict[str, bool] = (
+            json.loads(request.form["configOverwrite"])
+            if "configOverwrite" in request.form
+            else {}
+        )
```

Comment:
Maybe we should consolidate these two parameters? Something like:

```suggestion
        config_overwrite: Dict[str, bool] = (
            json.loads(request.form["configOverwrite"])
            if "configOverwrite" in request.form
            else {}
        )
        if request.form.get("overwrite") == "true":
            config_overwrite["dashboard"] = True
```

We'd then pass only `config_overwrite` to `ImportDashboardsCommand`. We could add the same logic to the other import commands, which would save us from having to define a custom `run` method in `superset/dashboards/commands/importers/v1/__init__.py`.